trending_data/src/main/webapp/unique_window_handler.js

```diff
@@ -22,6 +22,7 @@ export class UniqueWindowHandler {
   * to the popup
   */
   initPopup() {
+    this.showing = 'yt';
```

Comment:
It would be better to use some constants for this string to avoid problems in the future.